src/replication/replication_slave_node.cpp

```diff
@@ -97,30 +97,42 @@ namespace cubreplication
 	return error;
       }
 
+    cubstream::stream_position start_position = 0;
```

Comment:
setting start position code is duplicated - here and in replication_node_manager.cpp. can we avoid duplicating it?